api/StaticRange.json

```diff
@@ -11,7 +11,7 @@
             "version_added": "60"
           },
           "edge": {
-            "version_added": null
+            "version_added": true
```

Comment:
Confirmed by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6829 in Edge 18.